### PR TITLE
Change duration formatting in niceDuration

### PR DESCRIPTION
**Pull request by ddrone**
niceDuration used to display 61 second as "1:1" instead of "1:01". This patch fixes that and minute duration (which is 60 seconds last time I've checked) and adds couple of unit tests.

### CODE REVIEW

File: test/utils_test.dart [hunk 1]
```diff
@@ -82,5 +82,20 @@ true: bool"""));
 a: {}
 b: {}"""));
     });
+
+  });
+
+  group('niceDuration()', () {
+    test('formats duration longer than a minute correctly', () {
```
Comment (nex3) [inline test/utils_test.dart]: Also test a duration that's exactly on the minute boundary.

**Comment by nex3** [comment]
One additional test and I'll merge this in. Thanks for the fix!

File: test/utils_test.dart (hunk 1 above)
Comment (ddrone) [inline test/utils_test.dart]: Done

**Comment by ddrone** [comment]
Ping

**Comment by nex3** [comment]
Sorry, I was on vacation all last week.

**Comment by ddrone** [comment]
No problem!
